### PR TITLE
AppBar and CommandBar updates

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

- Adjust `AppBar` and `CommandBar` styles.
- Adjust the Stretch mode of `BitmapIcon` content
- Fix invalid Image size constraint

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarButtonTemplateSettings.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+namespace Windows.UI.Xaml.Controls.Primitives
+{
+	public  partial class AppBarButtonTemplateSettings : DependencyObject
+	{
+		public double KeyboardAcceleratorTextMinWidth
+		{
+			get
+			{
+				global::Windows.Foundation.Metadata.ApiInformation.TryRaiseNotImplemented("Windows.UI.Xaml.Controls.Primitives.AppBarButtonTemplateSettings", "KeyboardAcceleratorTextMinWidth");
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarButtonTemplateSettings.cs]: should this `TryRaiseNotImplemented` be removed?

File: src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarToggleButtonTemplateSettings.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+namespace Windows.UI.Xaml.Controls.Primitives
+{
+	public partial class AppBarToggleButtonTemplateSettings : global::Windows.UI.Xaml.DependencyObject
+	{
+		public double KeyboardAcceleratorTextMinWidth
+		{
+			get
+			{
+				global::Windows.Foundation.Metadata.ApiInformation.TryRaiseNotImplemented("Windows.UI.Xaml.Controls.Primitives.AppBarToggleButtonTemplateSettings", "KeyboardAcceleratorTextMinWidth");
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarToggleButtonTemplateSettings.cs]: ditto

File: src/Uno.UI/UI/Xaml/Controls/Image/Image.wasm.cs [hunk 3]
```diff
@@ -236,12 +236,12 @@ protected override Size MeasureOverride(Size availableSize)
 			// Always making sure the ret size isn't bigger than the available size for an image with a fixed width or height
 			ret = new Size(
 				!Double.IsNaN(Width) && (ret.Width > availableSize.Width) ? availableSize.Width : ret.Width,
-				!Double.IsNaN(Width) && (ret.Height > availableSize.Height) ? availableSize.Height : ret.Height
+				!Double.IsNaN(Height) && (ret.Height > availableSize.Height) ? availableSize.Height : ret.Height
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/Image/Image.wasm.cs]: oops!

File: src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarButtonTemplateSettings.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarButtonTemplateSettings.cs]: Those two really are not implemented.

Comment (davidjohnoliver) [reply src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarButtonTemplateSettings.cs]: should be marked `[NotImplemented]`, then

**Pull request merged**